astropy/visualization/wcsaxes/coordinate_helpers.py

```diff
@@ -460,6 +456,16 @@ def _draw(self, renderer, bboxes, ticklabels_bbox):
 
         renderer.close_group('coordinate_axis')
 
+    def _draw_ticks(self, renderer, bboxes, ticklabels_bbox):
+
+        renderer.open_group('coordinate_axis')
```

Comment:
You can just use names that describe what is being drawn, e.g. 'ticks' or 'grid lines'